templates/cassiopeia/html/layouts/chromes/cardGrey.php

```diff
@@ -21,34 +21,22 @@
 }
 
 $moduleTag              = $params->get('module_tag', 'div');
-$moduleAttribs          = [];
+$modulePos 		= $module->position;
 $moduleAttribs['class'] = $module->position . ' card card-grey ' . htmlspecialchars($params->get('moduleclass_sfx'), ENT_QUOTES, 'UTF-8');
+$modId 			= 'mod-' . $module->id;
```

Comment:
```suggestion
$modId                 = 'mod-' . $module->id;
```
See my previous review comment.

Reply:
This is weird, because when I view these files in my repository it looks all neat and tidy like this:

![line-up](https://user-images.githubusercontent.com/36987494/93668076-32037000-fa82-11ea-9387-4833b89b726e.jpg)

How do I change them and re-submit it without mucking up the PR?

Reply:
You can just apply my suggestions here on GitHub by using the corresponding buttons below the suggestions, then you pull changes so you have them in your local environment.

The reason why it looks ok for you locally is that tabs are shown as so and so many spaces depending on the editor, and so your editor may show it in a way so that it looks ok, and others may not.